firefox-ios/Client/Frontend/Browser/MainMenu/Redux/MainMenuMiddleware.swift

```diff
@@ -6,8 +6,31 @@ import Common
 import Redux
 import ToolbarKit
 import Account
+import Shared
 
 final class MainMenuMiddleware {
+    private struct Options {
```

Comment:
This could be a little bit more clearly named. Perhaps `TelemetryAction`?
Also, why `struct` vs `enum`? here?

Reply:
Will renamed it to TelemetryAction and yes, enum will be more than enough for this case.